README.md

```diff
@@ -55,31 +57,35 @@ On this project used [postgreSQL](https://www.postgresql.org/). To use this proj
 # files table
 
     CREATE TABLE files (
-      id SERIAL PRIMARY KEY,
-      path VARCHAR(100) NOT NULL,
-      displayname VARCHAR(100) NOT NULL,
-      type VARCHAR(100) NOT NULL,
-      creator_user_id INT REFERENCES users (id) NOT NULL
+        id SERIAL PRIMARY KEY,
+        path VARCHAR(100) NOT NULL,
+        displayname VARCHAR(100) NOT NULL,
+        type VARCHAR(100) NOT NULL,
+        folder_id INT REFERENCES folders (id) ON DELETE CASCADE,
+        creator_id INT REFERENCES users (id) NOT NULL
     );
   
 # shared-files table
 
     CREATE TABLE shared_files (
-      file_id INT REFERENCES files (id) NOT NULL,
-      user_id INT REFERENCES users (id) NOT NULL
+        file_id INT REFERENCES files (id) ON DELETE CASCADE NOT NULL,
+        user_id INT REFERENCES users (id) NOT NULL
```

Comment:
I think there should be a permission field here

Reply:
yes, I think so too